.prettierrc

```diff
@@ -5,5 +5,7 @@
     "printWidth": 150,
     "bracketSpacing": false,
     "endOfLine": "lf",
-    "tabWidth": 4
+    "tabWidth": 4,
+    "importOrder": ["", "^node:", "", "<THIRD_PARTY_MODULES>", "", "^zigbee", "", "^[./]"],
```

Comment:
Good call on the alt package.
I'd say let's take advantage of it:
```js
    importOrder: [
        "<TYPES>^(node:)",
        "",
        "<TYPES>",
        "",
        "<TYPES>^[.]",
        "",
        "<BUILTIN_MODULES>",
        "",
        "<THIRD_PARTY_MODULES>",
        "",
        "^zigbee",
        "",
        "^[.]",
    ],
```

Most will never be used, but at least it's nice and tidy.

Reply:
Done!